src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -38,7 +38,7 @@ public Game getGame(@PathVariable("gameId") String gameId){
     }
 
     @PostMapping(value = "/joinGame/{gameId}")
-    public Game joinGame(@PathVariable("gameId") String gameId, @RequestBody Player player) throws Exception{
+    public Game joinGame(@PathVariable("gameId") String gameId, @RequestBody Player player) throws Exception {
```

Comment:
added space after method (before brackets) for best practices and readability.
This is recommended by the "Spacing & line breaks" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3